src/content/configuration/dev-server.md

```diff
@@ -993,7 +993,11 @@ module.exports = {
 
 Sometimes you don't want to proxy everything. It is possible to bypass the proxy based on the return value of a function.
 
-In the function you get access to the request, response and proxy options. It must return either `false` or a path that will be served instead of continuing to proxy the request.
+In the function you get access to the request, response and proxy options.
+
+- Return `null` or `undefined` means continue to process the request with proxy.
```

Comment:
```suggestion
- Return `null` or `undefined` to continue processing the request with proxy.
```

Reply:
Does it also mean, returning nothing, would do the same as `null` and `undefined`? Maybe worth mentioning

Reply:
Return nothing is essentially return `undefined`. I think it is clear enough.